docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "ZDS versus ZGW"
+date: '11-01-2019'
+---
+
+De Zaak- en Documentservices (ZDS) wijken af van de Zaakgericht werken (ZGW)
+APIs, maar er is ook overlap. De belangrijkste verschillen en overeenkomsten op
+een rij:
+
+1. De ZGW APIs zijn in REST/JSON terwijl ZDS volgens SOAP/XML werken.
+2. De ZGW APIs zijn veel flexibeler dan ZDS. Dit is nodig omdat er geen directe
+   acties meer op de database mag worden gedaan door bijvoorbeeld een 
+   Zaaksysteem. Alle acties moeten via de ZGW APIs verlopen.
+3. De ZGW APIs kunnen alles wat ZDS ook kan.
+4. De ZGW APIs zijn opgedeeld. Er is niet meer 1 specificatie of endpoint om
+   alle operaties, zoals deze in ZDS bestonden, uit te voeren.
+5. De ZGW APIs zullen sneller opvolgende versies krijgen. Hierbij is het niet
+   ondenkbaar dat de Zaken API op versie 2 zit, terwijl de Besluiten API al op
+   versie 4 zit. Het is daarom ook niet zo dat ZGW een versie nummer krijgt.
+   Immers is ZGW alleen maar de verzamelnaam van alle Zaakgericht werken APIs. 
+6. De ZGW APIs worden aangetoond middels een referentie implementatie om zo 
```

Comment:
En om het vereiste gedrag te definieren daar waar niet in de specificatie kan worden vastgelegd.